Source/WebCore/platform/DateComponents.cpp

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2009 Google Inc. All rights reserved.
+ * Copyright (C) 2023 Apple Inc. All rights reserved.
+ * Copyright (C) 2009-2013 Google Inc. All rights reserved.
```

Comment:
I don't think that removing code grants Google copyright over remaining code.

Reply:
Reverted back this Copyright change for Google but kept for Apple. Pushed in new commit. :-)

Reply:
I will remove this Copyright change once all EWS pass and then merge. 👍